tensorboard/plugins/hparams/tf_hparams_dashboard/tf-hparams-dashboard.html

```diff
@@ -17,21 +17,33 @@
 
 <link rel="import" href="../polymer/polymer.html">
 <link rel="import" href="../tf-hparams-main/tf-hparams-main.html">
+<link rel="import" href="../tf-hparams-backend/tf-hparams-backend.html">
 
 <dom-module id="tf-hparams-dashboard">
   <template>
-    <!-- Tensorboard doesn't specify an experimentName. Currently it only
+    <!-- Tensorboard does not specify an experimentName. Currently it only
          supports one experiment per invocation. -->
     <tf-hparams-main
       id="hparams-main"
-      api-url="data/plugin/hparams"
+      data/plugin/hparams
+      backend="[[_backend]]"
       experiment-name="">
     </tf-hparams-main>
   </template>
   <script>
     'use strict';
     Polymer({
       is: 'tf-hparams-dashboard',
+      properties: {
+        _backend: {
+          type: Object,
+          value: () => {
+            return new tf.hparams.Backend(
+                /* apiUrl= */ 'data/plugin/hparams',
```

Comment:
you can pass `tf.backend.getRouter().pluginRoute` if you can https://github.com/tensorflow/tensorboard/blob/master/tensorboard/components/tf_backend/router.ts#L45

Reply:
Done.